src/extension.ts

```diff
@@ -89,6 +89,8 @@ export async function activate(context: vscode.ExtensionContext) {
     )
     globals.outputChannel = toolkitOutputChannel
 
+    ExtensionUse.instance.updateIsFirstUse(context)
```

Comment:
this is pretty early, would it make sense to be conservative and only set this after the New Connections page successfully loaded?

Reply:
I figured it didn't hurt to do this early since this could be used in other places eventually and the earlier we initialized it, the less likely to run in to a race condition.